### ISSUE TITLE
A call to `redub test` produces test binary without executable permissions making the call fail on Ubuntu 24.04

### ISSUE THREAD

**Issue by nordlow** [MrcSnm/redub#28]
Reproduce on Ubuntu 24.04:

Doing `redub test`, for instance, inside a local clone of https://github.com/nordlow/gmp-d outputs

```
     Warning 
     Warning ## Warning for package redub ##
     Warning 
     Warning The following compiler flags have been specified in the package description
     Warning file. They are handled by DUB and direct use in packages is discouraged.
     Warning Alternatively, you can set the DFLAGS environment variable to pass custom flags
     Warning to the compiler, or use one of the suggestions below:
     Warning 
     Warning debugMode: Call DUB with --build=debug
     Warning debugInfo: Call DUB with --build=debug
     Warning 
Dependencies resolved - 0 ms "debug" using /home/per/.local/dlang/linux/bin64/dmd v2.110.0-beta.1-380-g50966e0232 [linux-x86_64] - no parallel
Up-to-Date: gmp-d; 
Up-to-Date: gmp-d, skipping linking
Finished gmp-d - 0ms - To force a rebuild of up-to-date targets, run again with --force
/bin/sh: 1: /home/per/Work/gmp-d/bin/gmp-d: Permission denied
Error Program exited with code 126
```

.

**Comment by MrcSnm** [comment]
Fixed in redub v1.14.9 with a bunch of new fixes to test

**Comment by nordlow** [comment]
Awesome.
